src/pykeen/training/lcwa.py

```diff
@@ -173,3 +174,7 @@ def _check_slicing_availability(self, supports_sub_batching: bool):
             )
         logger.warning(report)
         raise MemoryError("The current model can't be trained on this hardware with these parameters.")
+
+
+class AcceleratedLCWATrainingLoop(AcceleratedTrainingLoop, LCWATrainingLoop):
+    """Super-powered LCWA training loop."""
```

Comment:
This needs a better docstring :sweat_smile: 

In particular it should mention that this is a distributed data-parallel (?) version of the LCWA  training loop using the huggingface accelerate library.

Reply:
shhhhhh i wrote the code so this means someone else is responsible for the docs, right?

Reply:
done in 97c84f66